.k8s/templates/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "standard-web.fullname" . }}
```

Comment:
Some of these we might just be able to hard code, like `standard-web.fullname`, especially if you're going to implement one of these in each library. That may even allow you to get rid of the `_helpers.tpl` entirely. I'm not sure about that though.

If you want to break this out and just have one set of these configs that you can use across ocelot and lynx, then this generalization will be important.